modules/configuring-transparent-huge-pages.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.adoc
+
+:_content-type: PROCEDURE
+[id="disable-thp_{context}"]
+= How to configure Transparent Huge Pages
```

Comment:
Additionally, I think we should change the name of the file to `modules/disabling-transparent-huge-pages.adoc`.

Reply:
Makes sense, I also considered doing this.  The reason I did not, is that the user may specify "madvise" or "always" instead of "never" and then he/she is not disabling them.  Only the example below shows how to disable them (one specific case of configuration).  Nevertheless, using your suggestion then.